fritz

```diff
@@ -1,9 +1,86 @@
 #!/usr/bin/env python
 import argparse
-import sys
-import subprocess
+from contextlib import contextmanager
+import json
 import os
 from pathlib import Path
+import subprocess
+import sys
+
+
+@contextmanager
```

Comment:
If we start the file with:

```
try:
    import baselayer
except ImportError:
    ... clone baselayer
```

from baselayer import ...

Then we can re-use that functionality instead of duplicating it here.

Reply:
I agree with your point. I just wanted to minimize the number of dependencies for the Fritz utility itself (it's now only using the "stdlib" stuff), so just copied it over. We can actually remove that function for now, it's not used anyways.